/**:
  mqtt_bridge_node:
    ros__parameters:
      forward_speed: [0.75,1.5,3.0]
      turn_speed: [0.75,1.5,3.0]
      broker_ip: "127.0.0.1"
      broker_port: 1883
      passwd: ""
      reconnect_delay: 15